Mathlib/Algebra/Category/GroupCat/Adjunctions.lean

```diff
@@ -94,17 +101,26 @@ namespace GroupCat
 -/
 def free : Type u ⥤ GroupCat where
   obj α := of (FreeGroup α)
-  map X Y := FreeGroup.map
-  map_id' := by intros; ext1; rfl
-  map_comp' := by intros; ext1; rfl
+  map := FreeGroup.map
+  map_id := by
+    intros; ext1; rw [←FreeGroup.map.unique]; intros; rfl
+  map_comp := by
+    intros; ext1; rw [←FreeGroup.map.unique]; intros; rfl
 #align Group.free GroupCat.free
 
 /-- The free-forgetful adjunction for groups.
 -/
 def adj : free ⊣ forget GroupCat.{u} :=
   Adjunction.mkOfHomEquiv
     { homEquiv := fun X G => FreeGroup.lift.symm
-      homEquiv_naturality_left_symm := fun X Y G f g => by ext1; rfl }
+      homEquiv_naturality_left_symm := by
+        intros
+        ext1
+        simp only [Equiv.symm_symm]
+        apply Eq.symm
+        apply FreeGroup.lift.unique
+        intros
+        apply FreeGroup.lift.of }
```

Comment:
Please leave a porting note when simple proofs fail. Something like
```
-- Porting note: used to be just `by intros; ext1; rfl`.
```

Reply:
This applies in a few other places. We can proceed with merging with these regressions, but I'd like to have breadcrumbs so it possible to eventually try to resolve these issues.